### PR TITLE
Turkish language added

### PR DESCRIPTION
**Pull request by faisalahammad**

### CODE REVIEW

File: SignalAnalysis/localization/strings.tr.resx [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="strBtnAccept" xml:space="preserve">
+    <value>&amp;aktif</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:121]: Capital first letter
Aktif or aktif?

File: SignalAnalysis/localization/strings.tr.resx [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strBtnCancel" xml:space="preserve">
+    <value>&amp;iptal</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:124]: Capital first letter
Iptal or iptal?

File: SignalAnalysis/localization/strings.tr.resx [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+Bunun yerine varsayılan değerler kullanılacaktır.</value>
+  </data>
+  <data name="strErrorDeserializeTitle" xml:space="preserve">
+    <value>Hata</value>
+  </data>
+  <data name="strFileHeader00" xml:space="preserve">
+    <value>ErgoLux verileri</value>
+  </data>
+  <data name="strFileHeader01" xml:space="preserve">
+    <value>Sinyal Analizi verileri</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:174]: Could it be 'SinyalAnalizi verileri' (without the space between signal and analysis)?

Besides that, is it 'SinyalAnalizi' or 'SinyalAnaliz' (as you suggested in the open/save dialog filters (see comments below)?

File: SignalAnalysis/localization/strings.tr.resx [hunk 4]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Maksimum</value>
+  </data>
+  <data name="strFileHeader09" xml:space="preserve">
+    <value>Asgari</value>
+  </data>
+  <data name="strFileHeader10" xml:space="preserve">
+    <value>Fraktal boyut</value>
+  </data>
+  <data name="strFileHeader11" xml:space="preserve">
+    <value>fraktal varyans</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:204]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 5]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strFileHeader12" xml:space="preserve">
+    <value>yaklaşık entropi</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:207]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 6]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strFileHeader13" xml:space="preserve">
+    <value>Örnek entropi</value>
+  </data>
+  <data name="strFileHeader14" xml:space="preserve">
+    <value>Shannon entropisi</value>
+  </data>
+  <data name="strFileHeader15" xml:space="preserve">
+    <value>entropi biti</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:216]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 7]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strFileHeader16" xml:space="preserve">
+    <value>ideal entropi</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:219]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 8]
```diff
@@ -0,0 +1,522 @@
...
+    <value>'{0}' bölümü yanlış biçimlendirilmiş.</value>
+  </data>
+  <data name="strFrmLanguage" xml:space="preserve">
+    <value>Kültür seçin</value>
+  </data>
+  <data name="strFrmSettings" xml:space="preserve">
+    <value>Ayarlar</value>
+  </data>
+  <data name="strFrmTitle" xml:space="preserve">
+    <value>sinyal analizi</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:264]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 9]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strFrmTitleUnion" xml:space="preserve">
+    <value> - </value>
+  </data>
+  <data name="strGrpAxis" xml:space="preserve">
+    <value>apsis ekseni</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:270]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 10]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Dizi dizini sonu (maks {0})</value>
+  </data>
+  <data name="strLblSeries" xml:space="preserve">
+    <value>Seri seçin</value>
+  </data>
+  <data name="strLblStart" xml:space="preserve">
+    <value>Dizi indeksi başlangıcı</value>
+  </data>
+  <data name="strLblWindow" xml:space="preserve">
+    <value>pencere</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:291]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 11]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Durmak</value>
+  </data>
+  <data name="strOpenDlgFilter" xml:space="preserve">
+    <value>ErgoLux dosyaları (*.elux)|*.elux|SinyalAnaliz dosyaları (*.sig)|*.sig|Metin dosyaları (*.txt)|*.txt|İkili dosyalar (*.bin)|*.bin|Tüm dosyalar (*.*)|*.*</value>
+  </data>
+  <data name="strOpenDlgTitle" xml:space="preserve">
+    <value>Veri dosyasını aç</value>
+  </data>
+  <data name="strPlotAppliedTitle" xml:space="preserve">
+    <value>pencereli sinyal</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:356]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 12]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Büyüklük (RMS²)</value>
+  </data>
+  <data name="strPlotFFTYLabelPow" xml:space="preserve">
+    <value>Güç (dB)</value>
+  </data>
+  <data name="strPlotFractalDisributionXLabel" xml:space="preserve">
+    <value>Fraktal boyut (H)</value>
+  </data>
+  <data name="strPlotFractalDisributionYLabel" xml:space="preserve">
+    <value>olasılık</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:380]: Capital first letter?

File: SignalAnalysis/localization/strings.tr.resx [hunk 13]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Geçerli kültür biçimlendirmesi</value>
+  </data>
+  <data name="strRadInvariantCulture" xml:space="preserve">
+    <value>Değişmez kültür biçimlendirme</value>
+  </data>
+  <data name="strRadPoints" xml:space="preserve">
+    <value>Veri noktaları</value>
+  </data>
+  <data name="strRadSeconds" xml:space="preserve">
+    <value>saniye</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.tr.resx:425]: Capital first letter?

**Review by arthurits — COMMENTED** [review]
Thanks so much for your translation @faisalahammad!
Before merging your PR, I'd like you to check the comments.

File: SignalAnalysis/localization/strings.tr.resx (hunk 1 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:121]: Aktif

File: SignalAnalysis/localization/strings.tr.resx (hunk 2 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:124]: Iptal

File: SignalAnalysis/localization/strings.tr.resx (hunk 3 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:174]: SinyalAnalizi verileri

File: SignalAnalysis/localization/strings.tr.resx (hunk 4 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:204]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 5 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:207]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 6 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:216]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 7 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:219]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx [hunk 14]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strFileHeader26" xml:space="preserve">
+    <value>ve</value>
+  </data>
+  <data name="strFileHeader27" xml:space="preserve">
+    <value>milisaniye</value>
+  </data>
+  <data name="strFileHeaderSection" xml:space="preserve">
+    <value>'{0}' bölümü yanlış biçimlendirilmiş.</value>
```
[the remark this answers is not in the document]
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:255]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 8 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:264]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 9 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:270]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 10 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:291]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 11 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:356]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 12 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:380]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx (hunk 13 above)
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:425]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx [hunk 15]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Kümülatif fraktal boyutu hesaplayın (yoğun CPU)</value>
+  </data>
+  <data name="strStatusTipPower" xml:space="preserve">
+    <value>FFT güç spektrumunu (dB) veya büyüklüğünü çizin</value>
+  </data>
+  <data name="strTabGUI" xml:space="preserve">
+    <value>Kullanıcı arayüzü</value>
+  </data>
+  <data name="strTabPlot" xml:space="preserve">
+    <value>çizim</value>
```
[the remark this answers is not in the document]
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:481]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx [hunk 16]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strToolStripAbout" xml:space="preserve">
+    <value>Hakkında</value>
+  </data>
+  <data name="strToolStripExit" xml:space="preserve">
+    <value>çıkış</value>
```
[the remark this answers is not in the document]
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:487]: Yes, please

File: SignalAnalysis/localization/strings.tr.resx [hunk 17]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Hata</value>
+  </data>
+  <data name="strReadNotimplementedError" xml:space="preserve">
+    <value>Bir {0} dosyasından veri okunamıyor.</value>
+  </data>
+  <data name="strReadNotimplementedErrorTitle" xml:space="preserve">
+    <value>Uygulanmadı</value>
+  </data>
+  <data name="strSaveDlgFilter" xml:space="preserve">
+    <value>Metin dosyası (*.txt)|*.txt|SinyalAnaliz dosyası (*.sig)|*.sig|İkili dosya (*.bin)|*.bin|Sonuç dosyası (*.results)|*.results|Tüm dosyalar (*.*)|*.*</value>
```
[the remark this answers is not in the document]
Comment (faisalahammad) [reply SignalAnalysis/localization/strings.tr.resx:460]: Sinyal Analizi

**Comment by faisalahammad** [comment]
Hi @arthurits 
Thanks for your revisions. 

1. Yes, I've followed the English translation, not Bengali. 
2. Sure, I'm not worried about hacktoberfest. 
3. Ok, I'll try to contribute there too.

**Review by arthurits — APPROVED** [review]

**Comment by arthurits** [comment]
Thanks so much @faisalahammad for the Turkish translation!

Here's a screenshot of the translated application:

image

**Pull request merged**